internal/internal_workflow_testsuite_test.go

```diff
@@ -392,6 +413,30 @@ func testWorkflowHello(ctx Context) (string, error) {
 	return result, nil
 }
 
+func testWorkflowHelloWithDataConverter(ctx Context) (string, error) {
```

Comment:
can you move this function as workflowFn := func() ... into the Test_ActivityMockFunction_WithDataConverter